test/integration/golden_response/staging/get_triples/stats_var.json

```diff
@@ -312,7 +312,43 @@
       "objectTypes": [
         "Property"
       ],
-      "provenanceId": "dc/d7tbsb1"
+      "provenanceId": "dc/lly3yc1"
+    },
+    {
+      "subjectId": "Count_Person",
+      "predicate": "censusACSTableId",
```

Comment:
the censusACSTableId predicate is new, is this expected @pradh

Reply:
yes, it was a predicate before too, for generated DCIDs: https://datacommons.org/browser/dc/cv35g4gzkth83

Reply:
So this is expected? Then in prod it was filtered out in prophet..